src/node_crypto.cc

```diff
@@ -741,6 +743,64 @@ void SecureContext::AddCRL(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+void SecureContext::UseExtraCaCerts(const char* fname) {
+  extra_root_certs_file = fname;
+}
+
+static void emitWarning(Environment* env, const char* fmt, ...) {
+  char warning[1024];
+  va_list ap;
+
+  va_start(ap, fmt);
+  vsnprintf(warning, sizeof(warning), fmt, ap);
+  va_end(ap);
+
+#if 0
+  Local<Object> process = env->process_object();
+  Local<v8::Value> f = process->Get(
+    FIXED_ONE_BYTE_STRING(env->isolate(), "emitWarning"));
+  Local<Value> args[] = {
+    node::OneByteString(env->isolate(), warning, strlen(warning))
+  };
+  f.As<v8::Function>()->Call(process, arraysize(args), args);
+
+#else
+  Local<Object> process = env->process_object();
+  Local<Value> args[] = {
+    node::OneByteString(env->isolate(), warning, strlen(warning))
+  };
+  MakeCallback(env, process, "emitWarning", arraysize(args), args);
+#endif
+}
```

Comment:
This function needs a `HandleScope`, it's leaking handles into the scope of the caller.